src/sentry/receivers/sentry_apps.py

```diff
@@ -52,7 +52,12 @@ def send_issue_resolved_webhook(organization_id, project, group, user, resolutio
 @issue_ignored.connect(weak=False)
 def send_issue_ignored_webhook(project, user, group_list, **kwargs):
     for issue in group_list:
-        send_workflow_webhooks(project.organization, issue, user, "issue.ignored")
```

Comment:
For some time, we will need to send two times the webhooks. One as ignored and one as archived.

We may want to work on a plan to sunsetting ignored issues as part of v2.